ddtrace/appsec/processor.py

```diff
@@ -297,13 +299,21 @@ def on_span_finish(self, span):
             if info.errors:
                 span.set_tag_str(APPSEC_EVENT_RULE_ERRORS, json.dumps(info.errors))
             span.set_tag_str(APPSEC_EVENT_RULE_VERSION, info.version)
+            # telemetry_writer.add_gauge_metric(TELEMETRY_APPSEC, "event_rules.version", info.version, {})
```

Comment:
What's the plan with these commented lines?

Reply:
I added a TODO below:
https://github.com/DataDog/dd-trace-py/pull/5046/files/49c7b6f056188c30450615d1349b0165e161309a#diff-0b32c6cdae8bdd01786317bcabebf73dbb57e535adab74c94858fda05da411efR610

I'm going to add the same TODO here